src/Visualization/Python/PlotAlongLine.py

```diff
@@ -103,12 +103,11 @@ def plot_along_line_command(
     points.
     """
     # Interpolate the selected quantities to the line
-    dim = len(line_start)
     if line_start is None or line_end is None:
         raise click.UsageError(
             f"Specify '--line-start' / '-A' and '--line-end' / '-B'."
         )
-    if len(line_start) != dim or len(line_end) != dim:
```

Comment:
Is there a way to actually check that the start/end point has the correct number of components?

Reply:
Not without opening one of the h5 files again. You'll get a "mismatched dimensions" error from the interpolation function, which is not as nice as catching this earlier (here) but enough for now I think.